src/components/Badge.vue

```diff
@@ -54,25 +54,32 @@ $badge-border-radius: $border-radius - 1px !default;
   @include badge-variation(default);
 
   @include font-styles(badge);
-  display: inline-block;
+  display: flex;
```

Comment:
Changing this will have an unexpected impact in places where the badge is used in a place with lots of available width and the badge will grow to fill the space 😅

Is there a specific thing this was needed for? Maybe there's a different way to achieve it or perhaps we could use `inline-flex` instead?

Reply:
Fixed it by applying the change to the relevant badge only.